nilearn/_utils/data_gen.py

```diff
@@ -462,18 +463,18 @@ def write_fake_fmri_data_and_design(shapes,
     shapes : :obj:`list` of :obj:`tuple`s of :obj:`int`
         A list of shapes in tuple format.
 
-    rk : :obj:`int`, optional
+    rk : :obj:`int`, default=3
```

Comment:
Why are you changing that ? This is not our standard practice.

Reply:
It is now: https://nilearn.github.io/dev/development.html#coding-style

😉